pkg/controller/installation/errors.go

```diff
@@ -69,6 +69,10 @@ type RenderManifestError error
 
 type IncompleteReleaseError error
 
+type FetchChartError error
```

Comment:
This is likely not what you want, it's **not** type aliasing. We have a bunch of places like this in the code and they all should be fixed.

We have a custom build of staticcheck that catches this, please use it and consider adding it to CI.